### PR TITLE
add multilang support and update view

### PR DESCRIPTION
**Pull request by WinterSilence**

### CODE REVIEW

File: classes/Kohana/Date.php [hunk 1]
```diff
@@ -514,12 +514,12 @@ public static function fuzzy_span($timestamp, $local_timestamp = NULL)
 		if ($timestamp <= $local_timestamp)
 		{
 			// This is in the past
-			return $span.' ago';
+			return __($span).' '.__('ago');
```
Comment (birkir) [inline classes/Kohana/Date.php]: Shouldn't this be done with parameters?
__(':span ago', array(':span' => __($span)));
as some languages will have "ago", before the actual $span.

Comment (WinterSilence) [inline classes/Kohana/Date.php]: what would you suggest?

Comment (birkir) [inline classes/Kohana/Date.php]: http://kohanaframework.org/3.3/guide/kohana/files/i18n#i18n-variables
I wrote an example above.

Comment (WinterSilence) [inline classes/Kohana/Date.php]: ups :) not immediately understand you. make changes now. thanks!

**Pull request closed without merging**